habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -119,15 +119,22 @@ def __init__(
         else:
             self._n_input_depth = 0
 
+        if "semantic_category" in observation_space.spaces:
+            self._n_input_semantic_category = observation_space.spaces["semantic_category"].shape[2]
+            spatial_size = observation_space.spaces["semantic_category"].shape[0] // 2
+            assert self._n_input_semantic_category == 3, "ResNetEncoder only supports RGB values from SemanticCategory sensor!"
+        else:
+            self._n_input_semantic_category = 0
+
         if normalize_visual_inputs:
             self.running_mean_and_var: nn.Module = RunningMeanAndVar(
-                self._n_input_depth + self._n_input_rgb
+                self._n_input_depth + self._n_input_rgb + self._n_input_semantic_category
```

Comment:
normalizing semantic_category inputs doesn't look right.

Reply:
The semantic_category input is forced to be RGB (see earlier comment).